src/rgw/rgw_op.cc

```diff
@@ -754,7 +754,7 @@ int rgw_build_object_policies(rgw::sal::RGWRadosStore *store, struct req_state *
       
     s->object->set_atomic(s->obj_ctx);
     if (prefetch_data) {
-      s->object->set_prefetch_data(s->obj_ctx);
+      s->object->set_prefetch_data(s->obj_ctx, start, end);
```

Comment:
i'd rather we didn't involve the zipper RGWObject interface at all here. all we need from RGWObject is the object's key. this could just be:
```c++
  s->obj_ctx->set_prefetch_data(s->object->get_obj(), start, end);
```

cc @dang

Reply:
Done